eth/api_backend.go

```diff
@@ -275,6 +275,11 @@ func (b *EthAPIBackend) SendTx(ctx context.Context, signedTx *types.Transaction)
 	if locals := b.eth.localTxTracker; locals != nil {
 		locals.Track(signedTx)
 	}
+	// TODO(rjl493456442): If a transaction fails stateful validation (e.g., no
+	// available slot), an error will be returned to the user. However, locally
+	// submitted transactions may be resubmitted later via the local tracker, which
+	// could cause confusion that a previously "rejected" transaction is later
+	// accepted.
 	return b.eth.txPool.Add([]*types.Transaction{signedTx}, false)[0]
```

Comment:
So let's not return the error here!

Reply:
It makes sense to me

Reply:
Seems it's getting difficult to know a transaction has error or not...

Reply:
Okay. As localTxPool feature can be disabled by `--txpool.nolocals` flag 
please return error like now when the flag is on

Reply:
fixed